docs/reference/datatiers.asciidoc

```diff
@@ -1,100 +1,115 @@
 [role="xpack"]
 [[data-tiers]]
-=== Data tiers
-
-Common data lifecycle management patterns revolve around transitioning indices
-through multiple collections of nodes with different hardware characteristics in order
-to fulfil evolving CRUD, search, and aggregation needs as indices age. The concept
-of a tiered hardware architecture is not new in {es}.
-<<index-lifecycle-management, Index Lifecycle Management>> is instrumental in
-implementing tiered architectures by automating the managemnt of indices according to
-performance, resiliency and data retention requirements.
-<<overview-index-lifecycle-management, Hot/warm/cold>> architectures are common
```

Comment:
I believe we should still mention the hot/warm/cold architecture and how data tiers tie into it/drive the implementation further.